### PR TITLE
SO-5596: snapshot import with unpublished references

### PR DESCRIPTION
**Pull request by cmark**

### CODE REVIEW

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedImportApiTest.java [hunk 1]
```diff
@@ -653,10 +653,42 @@ public void import37NewOwlParentInSecondImportBatch() {
...
-	
-	private void validateBranchHeadtimestampUpdate(IBranchPath branch, String importArchiveFileName,
+  	
+  	@Test
+	public void import38SnapshotContentWithUnpublishedReferences() throws Exception {
+  		importAndValidateBranchHeadTimestampUpdate(branchPath, "SnomedCT_RF2Release_INT_20180223_content_w_and_wo_effective_time_mix.zip", false, Rf2ReleaseType.SNAPSHOT);
+	}
+  	
+  	@Test
+	public void import39SnapshotContentWithUnpublishedReferences() throws Exception {
```
Comment (AAAlinaaa) [inline snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedImportApiTest.java]: Shouldn't this be delta? The name is the same as above only the number is different.

**Comment by cmark** [comment]
7.x port PR available: #1159

**Review by AAAlinaaa — APPROVED** [review]
LGTM

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedImportApiTest.java (hunk 1 above)
Comment (cmark) [reply snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedImportApiTest.java]: Yes.

Comment (cmark) [reply snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedImportApiTest.java]: Also, the assertion requires one extra line to ensure that the import completes but reports an error.

**Review by nagyo — APPROVED** [review]
LGTM! 🍏

**Pull request merged**